index.js

```diff
@@ -111,12 +123,17 @@ app.use((err, req, res, next) => {
       console.debug("No token - redirect to login");
       return res.redirect(`/psychologue/login`);
     } else {
-      req.flash('error', "Cette page n'existe pas.")
+      req.flash('error', "Cette page n'existe pas.");
       return res.redirect(`/`);
     }
   } else if( req.cookies === undefined ) {
-    req.flash('error', "Cette page n'existe pas.")
+    req.flash('error', "Cette page n'existe pas.");
     res.redirect('/');
+  } else if (err !== 'EBADCSRFTOKEN') { // handle CSRF token errors here
+    console.warn(`CSRF token errors detected ${req.csrfToken()} but have ${res.req.body._csrf}`);
+    res.status(403);
+    req.flash('error', "Il semblerait que vous n'êtes pas à l'origine de cette opération, vous avez été redirigé.")
```

Comment:
message peut être bizarre

Reply:
je ne comprend pas bien ce qui déclenche cette erreur :/

Reply:
on peut faire une erreur plus generique, pour ne pas donner d'indices à un attaquant (vu que de toutes facons un user "normal" n'y comprend rien)
Genre "L'opération a échoué. Vous pouvez réessayer." (c'est assez pourri si tu trouves mieux je t'en prie :) )

Reply:
c'est cette erreur décrite ici "The server rejects the request if the token is invalid."
* https://github.com/pillarjs/understanding-csrf#csrf-tokens

Reply:
j'ai mis
> Une erreur est survenue, pouvez-vous réssayer ?